R/fd_inventory.R

```diff
@@ -124,6 +130,9 @@ fd_dendro <- function() {
   names(df)[names(df) == "subplot"] <- "subplot_id"
   names(df)[names(df) == "bands_in"] <- "band_in"
 
+  #sort by tag
```

Comment:
Be consistent, please add a space: `# sort by tag`

Reply:
got it, sorry. will do !